_targets.R

```diff
@@ -2,22 +2,61 @@ library(targets)
 source("1_fetch/src/get_nwis_data.R")
 source("2_process/src/process_and_style.R")
 source("3_visualize/src/plot_timeseries.R")
+tar_option_set(debug = "site_data")
 
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+####            p1_targets_list
+site_nums = c("01427207", "01432160", "01435000", "01436690", "01466500")
+####
+
+####            p3_targets_list
+p_width <- 12
+p_height <- 7
+p_units <- "in"
+####
+
+
 p1_targets_list <- list(
-  tar_target(
-    site_data,
-    download_nwis_data(),
+                        # using separate functions for each target
+tar_target(
+  nwis_01427207_data_csv,
+  download_nwis_site_data(filepath = '1_fetch/out/nwis_01466500_data.csv'),
```

Comment:
The site IDs in the target name and the filepath don't match for several of these targets.